packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -14,10 +9,7 @@
   // https://github.com/necolas/normalize.css/issues/566
   // https://github.com/necolas/normalize.css/commit/1be44f3b2a79af5d463c7ea0264fee0e6fa464ca
   // stylelint-disable selector-no-qualifying-type
-  button.button,
-  html input[type='button'].button,
-  input[type='reset'].button,
-  input[type='submit'].button {
```

Comment:
Since we only need `button[type="button"]`, we can get rid of some of the normalize styles.